pennylane/transforms/split_non_commuting.py

```diff
@@ -288,26 +288,15 @@ def circuit(x):
             batch_size=tape.batch_size,
         )
 
-    if (
-        grouping_strategy == "wires"
-        or grouping_strategy == "default"
-        and any(
-            isinstance(m, ExpectationMP) and isinstance(m.obs, LinearCombination)
-            for m in tape.measurements
-        )
-        or any(
-            m.obs is not None and not qml.pauli.is_pauli_word(m.obs) for m in single_term_obs_mps
-        )
+    if grouping_strategy == "wires" or any(
+        m.obs is not None and not qml.pauli.is_pauli_word(m.obs) for m in single_term_obs_mps
     ):
-        # This is a loose check to see whether wires grouping or qwc grouping should be used,
```

Comment:
The premise of the old logic here no longer holds, the graph-based grouping method consistently outperforms wire-based grouping after a refactor of the grouping module completed a few months ago, so there is no reason to use wire-based grouping as a default anymore.

Reply:
Should we make this change in an independent PR?  Given the premise for this PR is "tests" and not "performance improvement for grouping".

Reply:
The thing is, if we do not make the change here, we would have to change the tests to expect the old behavior, and when we do make the change, update the tests back to expect the new behavior.